Tests/ScipioKitTests/IntegrationTests.swift

```diff
@@ -154,8 +156,25 @@ final class IntegrationTests: XCTestCase {
             )
 
             for destination in expectedDestinations {
-                let frameworkRoot = xcFrameworkPath
+                let sdkRoot = xcFrameworkPath
                     .appendingPathComponent(destination)
+
+                if let buildOption = buildOptionsMatrix[frameworkName],
+                   buildOption.isDebugSymbolsEmbedded == true,
```

Comment:
```suggestion
                   buildOption.isDebugSymbolsEmbedded,
```

It's not an optional

Reply:
hi, looks like this is optional.

https://github.com/giginet/Scipio/blob/05e82383e64b9d3ad59e0650ee399045a294b1cf/Sources/ScipioKit/Runner.swift#L164

Reply:
Oops. Never mind!